lib/js/atoms/Stepper.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+	<div class="stepper">
+		<div v-for="(step, index) in steps" :key="index" class="stepper__item">
+			<div
+				v-if="index !== 0"
+				class="stepper__separator"
+				:class="{ '-active': step.isActive }"
+			></div>
+			<div class="stepper__icon" :class="{ '-active': step.isActive }">
+				<icon :icon="ICONS[step.iconKey]" :size="ICON_SIZES.X_SMALL"></icon>
```

Comment:
On designs I see different icon for steps that are finished. How do we want to address that?

Reply:
as above, in my opinion it's application specific logic, so it should not be in DS